samples/nrf9160/http_update/application_update/Kconfig

```diff
@@ -14,12 +14,18 @@ config DOWNLOAD_HOST
 	  of this configuration should be 'foo.bar'
 
 config DOWNLOAD_FILE
-	string "Application update file name"
+	string "Application update file name (version 2)"
 	help
-	  File part of URL to application update binary. For example
+	  File part of URL to application update binary version 2. For example
 	  if the file is stored in http://foo.bar/update.bin the value
 	  of this configuration should be 'update.bin'
 
+config DOWNLOAD_DOWNGRADE_FILE
+	string "Application update file name (version 1)"
```

Comment:
```suggestion
	string "Application downgrade file name (version 1)"
```

Reply:
I'm fine with calling both "update" and then having version 1 and 2, but then you should rename the name of the option to `DOWNLOAD_FILE_VERSION_1` or something.

Reply:
Renamed to DOWNLOAD_FILE_V1 / DOWNLOAD_FILE_2